packages/p2p-v2/src/components/TextArea/TextArea.scss

```diff
@@ -8,27 +8,58 @@
         height: 9.6rem;
         border-radius: 0.4rem;
         border: 1px solid #d6dadb;
-        outline-color: #85acb0;
+        outline: none;
         padding: 1rem 1.6rem;
         font-size: 1.4rem;
+        font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
Do we need to put font-family here?

Reply:
I just wanted to set it explicitly, but I'll remove it